### PR TITLE
playwright module unit tests

### PR DESCRIPTION
**Pull request by mnovitskyi-cap**

### CODE REVIEW

**Review by klaudia-nowak — APPROVED** [review]

**Comment by KMariusz** [comment]
@mnovitskyi-cap
1. Pull latest develop changes into your branch
2. Execute test cases
3. Attach evidences (passed result screen would be enough)

**Comment by mnovitskyi-cap** [comment]
pw-tests
All playwright tests

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/core/newDrivers/DriverManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.capgemini.mrchecker.playwright.core.newDrivers;
+
+
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/core/newDrivers/DriverManagerTest.java:4]: Double newline, please use formatter

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/tags/UnitTest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.Tag;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Tag("UnitTest")
+public @interface UnitTest {
+}
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/tags/UnitTest.java]: Missing newline at EOF

File: mrchecker-framework-modules/mrchecker-playwright-module/src/resources/settings.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
-playwright.browser.proxy=
-playwright.browser.channel=
+#playwright.browser.proxy=
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/resources/settings.properties]: Do we need this change? Are tests fail if you leave it assigned empty?

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/unit/DefaultDriverBrowserTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		TestPage testPage = PageFactory.getPageInstance(TestPage.class);
+		testPage.load();
+		String driverName = DriverManager.getDriver()
+				.browser()
+				.browserType()
+				.name();
+		assertEquals("chromium", driverName, "Wrong browser name");
+	}
+	
+}
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/unit/DefaultDriverBrowserTest.java]: Missing newline at EOF

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/unit/base/runtime/RuntimeParametersTest.java [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+		assertThat(browserOptionsValues, is(nullValue()));
+	}
+	
+	@Test
+	public void shouldToStringReturnKeyValue() {
+		String toStringValue = RuntimeParametersPlaywright.BROWSER.toString();
+		
+		assertThat(toStringValue, is(equalTo("browser=" + STARTUP_PARAMETERS_VALUES.get("browser"))));
+	}
+}
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/java/com/capgemini/mrchecker/playwright/unit/base/runtime/RuntimeParametersTest.java]: Missing newline at EOF

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/resources/allure.properties [hunk 6]
```diff
@@ -0,0 +1 @@
+allure.results.directory=target/allure-results
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/resources/allure.properties]: Missing newline at EOF

File: mrchecker-framework-modules/mrchecker-playwright-module/src/test/resources/junit-platform.properties [hunk 7]
```diff
@@ -0,0 +1,10 @@
+junit.jupiter.extensions.autodetection.enabled=true
+junit.jupiter.execution.parallel.enabled=true
+#junit.jupiter.execution.parallel.mode.default=concurrent
+junit.jupiter.execution.parallel.mode.default=same_thread
+junit.jupiter.execution.parallel.mode.classes.default=concurrent
+#junit.jupiter.execution.parallel.mode.classes.default=same_thread
+junit.jupiter.execution.parallel.config.strategy=dynamic
+junit.jupiter.execution.parallel.config.dynamic.factor=1
+cucumber.ansi-colors.disabled=false
+cucumber.publish.quiet=true
```
Comment (KMariusz) [inline mrchecker-framework-modules/mrchecker-playwright-module/src/test/resources/junit-platform.properties]: Missing newline at EOF
